minesweeper.py

```diff
@@ -320,9 +331,11 @@ def new_game(event=None):
 if root.tk.call("tk", "windowingsystem") == "aqua":
     top_menu.add_cascade(label="Game", menu=top_menu_game)
     top_menu_game.add_command(label="New Game", accelerator="F2", command=new_game)
+    top_menu_game.add_command(label="High Score", accelerator="F6", command=highscore_window)
```

Comment:
Maybe `"High Scores"` with `s` at the end, and similarly for all other places where `"High Score"` is mentioned?